src/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.csproj

```diff
@@ -12,6 +12,8 @@
   </ItemGroup>
```

Comment:
I am hitting a failure in CI, but only on Windows.  Any idea how I fix this?
```
Error : Multiple projects built twice with the same target path D:\j\workspace\windows_nt_de---0cbbd14c\bin\ref\System.Runtime\4.1.1.0\System.Runtime.dll.
D:\j\workspace\windows_nt_de---0cbbd14c\build.proj -->
  Global Properties:
    ConfigurationGroup = Debug
    OSGroup = Windows_NT
    WithoutCategories = IgnoreForCI
D:\j\workspace\windows_nt_de---0cbbd14c\src\dirs.proj -->
  Global Properties:
    ConfigurationGroup = Debug
    WithoutCategories = IgnoreForCI
    DefaultBuildAllTarget = Build
    BuildAllProjects = true
    InputOSGroup = Windows_NT
    FilterToOSGroup = Windows_NT
D:\j\workspace\windows_nt_de---0cbbd14c\src\src.builds -->
  Global Properties:
    ConfigurationGroup = Debug
    WithoutCategories = IgnoreForCI
    DefaultBuildAllTarget = Build
    BuildAllProjects = true
    InputOSGroup = Windows_NT
    FilterToOSGroup = Windows_NT
D:\j\workspace\windows_nt_de---0cbbd14c\src\System.Runtime.Serialization.Primitives\src\System.Runtime.Serialization.Primitives.builds -->
  Global Properties:
    ConfigurationGroup = Debug
    WithoutCategories = IgnoreForCI
    DefaultBuildAllTarget = Build
    BuildAllProjects = true
    InputOSGroup = Windows_NT
    FilterToOSGroup = Windows_NT
D:\j\workspace\windows_nt_de---0cbbd14c\src\System.Runtime.Serialization.Primitives\src\System.Runtime.Serialization.Primitives.csproj -->
  Global Properties:
    ConfigurationGroup = Debug
    WithoutCategories = IgnoreForCI
    BuildAllProjects = true
    InputOSGroup = Windows_NT
    OSGroup = Windows_NT
    TargetGroup = netcore50aot
D:\j\workspace\windows_nt_de---0cbbd14c\src\System.Runtime\ref\System.Runtime.csproj
  Global Properties:
    ConfigurationGroup = Debug
    WithoutCategories = IgnoreForCI
    BuildAllProjects = true
    InputOSGroup = Windows_NT
    OSGroup = Windows_NT
    TargetGroup = netcore50aot

D:\j\workspace\windows_nt_de---0cbbd14c\build.proj -->
  Global Properties:
    ConfigurationGroup = Debug
    OSGroup = Windows_NT
    WithoutCategories = IgnoreForCI
D:\j\workspace\windows_nt_de---0cbbd14c\src\dirs.proj -->
  Global Properties:
    ConfigurationGroup = Debug
    WithoutCategories = IgnoreForCI
    DefaultBuildAllTarget = Build
    BuildAllProjects = true
    InputOSGroup = Windows_NT
    FilterToOSGroup = Windows_NT
D:\j\workspace\windows_nt_de---0cbbd14c\src\src.builds -->
  Global Properties:
    ConfigurationGroup = Debug
    WithoutCategories = IgnoreForCI
    DefaultBuildAllTarget = Build
    BuildAllProjects = true
    InputOSGroup = Windows_NT
    FilterToOSGroup = Windows_NT
D:\j\workspace\windows_nt_de---0cbbd14c\src\System.Collections\src\System.Collections.builds -->
  Global Properties:
    ConfigurationGroup = Debug
    WithoutCategories = IgnoreForCI
    DefaultBuildAllTarget = Build
    BuildAllProjects = true
    InputOSGroup = Windows_NT
    FilterToOSGroup = Windows_NT
D:\j\workspace\windows_nt_de---0cbbd14c\src\System.Collections\src\System.Collections.csproj -->
  Global Properties:
    ConfigurationGroup = Debug
    WithoutCategories = IgnoreForCI
    BuildAllProjects = true
    InputOSGroup = Windows_NT
D:\j\workspace\windows_nt_de---0cbbd14c\src\System.Runtime\src\System.Runtime.csproj -->
  Global Properties:
    ConfigurationGroup = Debug
    WithoutCategories = IgnoreForCI
    BuildAllProjects = true
    InputOSGroup = Windows_NT
D:\j\workspace\windows_nt_de---0cbbd14c\src\System.Runtime\ref\System.Runtime.csproj
  Global Properties:
    ConfigurationGroup = Debug
    WithoutCategories = IgnoreForCI
    BuildAllProjects = true
    InputOSGroup = Windows_NT


MSBUILD : error MSB4015: The build stopped unexpectedly because the "ReusableLogger" logger failed unexpectedly during shutdown.
System.Exception: 1 bin clashes were detected during the build.
   at Microsoft.DotNet.Build.Tasks.BinClashLogger.Shutdown()
   at Microsoft.Build.Evaluation.ProjectCollection.ReusableLogger.Shutdown()
   at Microsoft.Build.BackEnd.Logging.LoggingService.ShutdownLogger(ILogger logger)
```